fog-updater/src/fog_update.py

```diff
@@ -0,0 +1,252 @@
+#!/usr/bin/env python3
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+from github import Github, GithubException, InputGitAuthor, enable_console_debug_logging
+import datetime
+import io
+import os
+import requests
+import sys
+
+DEFAULT_ORGANIZATION = "badboy"
+DEFAULT_AUTHOR_NAME = "data-updater"
+DEFAULT_AUTHOR_EMAIL = "jrediger@mozilla.com"
+USAGE = "usage: fog-update"
+REVIEWERS = ["@mozilla/glean"]
+INDEX_URL = "https://raw.githubusercontent.com/mozilla/gecko-dev/master/toolkit/components/glean/metrics_index.py"  # noqa
+BODY_TEMPLATE = """
+This (automated) patch updates the list from metrics_index.py.
+
+For reviewers:
+
+* Canonical source for the index: <https://searchfox.org/mozilla-central/source/toolkit/components/glean/metrics_index.py>
+* Please double-check that the changes here are valid and that the referenced files exist.
```

Comment:
It's good to include consequences. "If the referenced files do not exist, schema deploys will fail" (though I suppose the CI run on the PR will fail first?